### DISCUSSION TITLE
SOC Option to Resolve IP to Hostname

### DISCUSSION THREAD

**Discussion by corycandia** [Security-Onion-Solutions/securityonion#5153] [Unsupported Versions] [answered]
Community,

Does anyone know if there is an option to have the SOC resolve IP addresses?  This would be useful for both internal DNS hosts and possibly public hosts if desired.

I have checked with the GUI settings, but not any internal config files on CLI.

If this is not a capability, is there a feature request page it should be requested for consideration?

**Comment by weslambert** [comment, marked as answer]
This is not currently a built-in capability, but you could do this in the Elastic pipeline if you wanted.
